src/lib/ProductExporter.ts

```diff
@@ -76,20 +76,50 @@ export class ProductExporter {
 
     private async retrieveProduct(conn: Connection, productList: Set<string>) {
         Util.showSpinner('products export');
+        
         let productDefinitions = await Queries.queryProduct(conn, productList);
```

Comment:
może zamiast wykonywać tę funkcję dla każdego obiektu można by ją tak zmienić żeby wykonywać ją raz dla całego obiektu z produktem, pricebookiem itd. Wtedy zredukujesz liczbę wykonań z 20+ do 5-6. Jeśli byłby z tym jakiś problem to może zostać tak jak jest teraz.

Reply:
Celowo tak zrobiłem, żeby error wyświetlał się po eksporcie danego obiektu - użytkownik będzie mógł łatwiej zidentyfikować, który eksport mu nie poszedł.